Doc/whatsnew/3.10.rst

```diff
@@ -1919,6 +1919,11 @@ Porting to Python 3.10
   instead.
   (Contributed by Victor Stinner and Erlend E. Aasland in :issue:`43908`.)
 
+* The undocumented function ``Py_FrozenMain`` has been removed from the
+  limited API. The function is mainly useful for custom builds of Python,
+  and it is not always present when Python is built as a shared library.
```

Comment:
"when Python is built as a shared library" the symbol is exported, do you mean the opposite? Maybe omit this sentence since I would like this issue to be fixed :-D

Reply:
OK, I'll remove it. But the fix could reword this, too :)